easybuild/easyconfigs/s/SPH-EXA/SPH-EXA-v0.2.2-gompi-2021a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "CMakeMake"
+
+name = "SPH-EXA"
+version = "v0.2.2"
+
+homepage = "https://github.com/unibas-dmi-hpc/SPH-EXA_mini-app"
+description = """
+The SPH-EXA mini-app is a C++ scientific research code; its goal is to scale
+the Smoothed Particle Hydrodynamics (SPH) method to enable Tier-0 and Exascale
+simulations. More infos here: https://hpc.dmi.unibas.ch/en/research/sph-exa/
+"""
+
+toolchain = {"name": "gompi", "version": "2021a"}
+
+source_urls = [
+    "https://github.com/unibas-dmi-hpc/SPH-EXA_mini-app/archive/refs/tags"]
+sources = ["%(version)s.tar.gz"]
+checksums = [
+    "1e4c371d6783cc46698c4e3ed9ddc1c67236bb3e57da22cf16693954b69ac77f"]
+
+builddependencies = [
+    ("CMake", "3.20.1", "", ("GCCcore", "10.3.0")),
```

Comment:
Don't hardcode the subtoolchain here please:

```python
builddependencies = [
    ('CMake', '3.20.1'),
]

Reply:
https://github.com/easybuilders/easybuild-easyconfigs/pull/13244/commits/16d74b36396d706e5d35d6a0053b74df46c35876